internal/cli/config.go

```diff
@@ -82,6 +86,14 @@ func (opts *configOpts) Run() error {
 				Help:    helpLink,
 			},
 		},
+		{
+			Name: "projectId",
+			Prompt: &survey.Input{
+				Message: "Project ID [You can leave it blank but remember to use the flag --projectId]:",
```

Comment:
let's shorter this to something like
```suggestion
				Message: "Project ID [optional]:",
```